lab05/src/mc322/lab05/Dama.java

```diff
@@ -18,15 +18,30 @@ public char presentPiece() {
     		return 'P';
     }
 
-    boolean isValid(){
+    boolean isValid(int []source, int []target, boolean comer){
         boolean valid = false;
-
+        boolean compatible;
+        int y, x;
+        y = Math.abs(source[0] - target[0]);
+        x = Math.abs(source[1] - target[1]);
+        compatible = isCompatible(y, x);

```

Comment:
A definição da função é isCompatible(int x, int y). Então, a chamada deveria ser isCompatible(x, y) e não isCompatible(y, x). No caso, não faria diferença já que é compatível se x == y, mas é melhor corrigir.
```suggestion
        compatible = isCompatible(x, y);

```